packages/wonder-blocks-modal/components/focus-trap.js

```diff
@@ -113,34 +130,49 @@ export default class FocusTrap extends React.Component<{children: React.Node}> {
             return;
         }
 
-        const modalRoot = this._getModalRoot();
+        const modalRoot = this.modalRoot;
         if (modalRoot.contains(target)) {
             // If the newly focused node is inside the modal, we just keep track
             // of that.
-            this._lastNodeFocusedInModal = target;
+            this.lastNodeFocusedInModal = target;
         } else {
             // If the newly focused node is outside the modal, we try refocusing
             // the first focusable node of the modal. (This could be the user
             // pressing Tab on the last node of the modal, or focus escaping in
             // some other way.)
-            this._focusFirstElementIn(modalRoot);
+            this.focusFirstElementIn(modalRoot);
 
             // But, if it turns out that the first focusable node of the modal
             // was what we were previously focusing, then this is probably the
             // user pressing Shift-Tab on the first node, wanting to go to the
             // end. So, we instead try focusing the last focusable node of the
             // modal.
-            if (document.activeElement === this._lastNodeFocusedInModal) {
-                this._focusLastElementIn(modalRoot);
+            if (document.activeElement === this.lastNodeFocusedInModal) {
+                this.focusLastElementIn(modalRoot);
             }
 
             // Focus should now be inside the modal, so record the newly-focused
             // node as the last node focused in the modal.
-            this._lastNodeFocusedInModal = document.activeElement;
+            this.lastNodeFocusedInModal = document.activeElement;
         }
     };
 
     render() {
-        return this.props.children;
+        return (
+            <View>
```

Comment:
It seems like this could be a React.Fragment, correct?

Reply:
Glad that we've moved this logic to be inside of the FocusTrap! We may want to consider exposing FocusTrap publicly, I think people would find it to be really useful.

Reply:
I thought about it.  It felt weird, but that's such a squishy argument.  It's not like we're relying on the layout properties of the View as a container in this case.